src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/RuntimeImports.cs

```diff
@@ -487,9 +487,63 @@ internal enum RuntimeHelperKind
             CheckArrayElementType,
         }
 
-        [MethodImplAttribute(MethodImplOptions.InternalCall)]
-        [RuntimeImport(RuntimeLibrary, "RhGetRuntimeHelperForType")]
-        internal static extern unsafe IntPtr RhGetRuntimeHelperForType(EETypePtr pEEType, RuntimeHelperKind kind);
+        internal static unsafe IntPtr RhGetRuntimeHelperForType(EETypePtr pEEType, RuntimeHelperKind kind)
```

Comment:
Instead of inlining this into RuntimeImports.cs, could you put the implementation as a `RuntimeExport` into src\coreclr\nativeaot\Runtime.Base\src\System\Runtime\RuntimeExports.cs? Use `EEType*` instead of `EETypePtr` there.

RuntimeHelperKind enum will have to move to src\coreclr\tools\Common\Internal\Runtime\RuntimeConstants.cs but that's a better spot for it.

Then you can keep this InternalCall as-is.